etc/testing/travis.sh

```diff
@@ -59,6 +59,8 @@ for i in $(seq 3); do
     sleep 10
 done
 
+pachctl config update context `pachctl config get active-context` --pachd-address=$(minikube ip):30650
```

Comment:
@msteffen This will enable direct connect for all tests. Do you know of any issue with this (i.e. are there any tests that expect to be using port forwarding)? If not, this should speed things up a good deal as a nice side effect.

Reply:
That's great! When testing locally I have always set `PACHD_ADDRESS` so I think this should work just based on that. In addition, though, I think in travis we run minikube with `--vm-driver=none`, so that there's no separate minikube VM (in other words, I think `minikube ip` yields `127.0.0.1`) so that even non-port-forwarded traffic sent to localhost should still arrive.